ecs_task_definitions.tf

```diff
@@ -1,13 +1,19 @@
 
 resource "aws_ecs_task_definition" "service" {
   family                   = var.service_name
-  container_definitions    = "[${module.service_container_definition.json_map_encoded}]"
+  container_definitions    = var.container_definition_json != null ? "[${var.container_definition_json}]" : "[${module.service_container_definition.json_map_encoded}]"
```

Comment:
If we're going to basically allow passthroughs here, I feel like we might as well support multiple container definitions too.

Reply:
I was literally thinking that as I pushed this PR, I'll update it